core/lib/tasks/solidus/import_existing_permission_sets.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+namespace :solidus do
+  desc "Import existing permission sets to role permissions table"
+  task import_existing_permission_sets: :environment do
+    Spree::PermissionSets::Base.subclasses.each do |permission|
+      Spree::PermissionSet.find_or_create_by(name: permission.to_s.split('PermissionSets::').last, set: permission.to_s)
```

Comment:
Nit: what do you think about downcasing the name and set?

Also, I'm not sure to understand what `set` is going to be used for. @the-krg or maybe @rainerdema, can you help me understand? :pray: